Sources/Db/APIs/MySQL.php

```diff
@@ -242,6 +242,30 @@ public function query(string $identifier, string $db_string, array $db_values =
 
 		$ret = @mysqli_query($connection, $db_string, self::$unbuffered ? MYSQLI_USE_RESULT : MYSQLI_STORE_RESULT);
 
+		if ($ret === false && empty($db_values['db_error_skip'])) {
+			list($file, $line) = $this->error_backtrace('', '', 'return', __FILE__, __LINE__);
+			$query_error = $this->error();
+
+			// Nothing's defined yet... just die with it.
+			if (empty(Utils::$context) || empty(Lang::$txt))
+				die($query_error);
```

Comment:
```suggestion
			if (empty(Utils::$context) || empty(Lang::$txt)) {
				die($query_error);
			}
```

Keep PHP-CS-Fixer happy